docs/topics/api/shelves.rst

```diff
@@ -97,11 +97,12 @@ Current implementation relies on Adzerk to determine which addons are returned a
     :query string lang: Activate translations in the specific language for that query. (See :ref:`translated fields <api-overview-translations>`)
     :query int page_size: specify how many addons should be returned.  Defaults to 6.  Note: fewer addons could be returned if there are fewer than specifed sponsored addons currently, or the Adzerk service is unavailable.
     :query string wrap_outgoing_links: If this parameter is present, wrap outgoing links through ``outgoing.prod.mozaws.net`` (See :ref:`Outgoing Links <api-overview-outgoing>`)
-    :>json array results: The array containing the addon results for this query.  The object is a :ref:`add-on <addon-detail-object>` as returned by :ref:`add-on search endpoint <addon-search>` with extra fields of ``click_url`` and ``click_data``
-    :>json string results[].click_url: the url to ping if the sponsored addon's detail page is navigated to.
-    :>json string results[].click_data: the signed data payload to send to ``click_url`` that identifies the sponsored placement clicked on.
+    :>json array results: The array containing the addon results for this query.  The object is a :ref:`add-on <addon-detail-object>` as returned by :ref:`add-on search endpoint <addon-search>` with an extra field of ``events``
+    :>json object results[].events: contains data that for different events that can be recorded.
```

Comment:
Would it make more sense to call this `event_data` than `events`?

Reply:
🤷‍♂️ https://github.com/mozilla/addons-server/pull/15768/commits/9896906dd5f3bc342a3de40da36c12984ff4bce4